public/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <title>Menu Service Component</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans|Open+Sans|Questrial&display=swap" rel="stylesheet">
```

Comment:
You imported all 3 fonts, but you can take out Open Sans and Questrial since we're not using them. I believe they actually make some diffference in the loading time.

Reply:
Oh, nice... I'll change that now. Thanks!